### PR TITLE
Refactor/543

### PR DESCRIPTION
**Pull request by Haytham1337**
## Description

Refactored registration form, added google sign in and fixed validation.
image
image

image

#### Screenshots

Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date admin and back-end part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

**Comment by Haytham1337** [comment]
fxing code smells SH

**Comment by dmytro07** [comment]
Please, link your issue
linkissue

File: src/pages/login/login.js [hunk 1]
```diff
@@ -65,16 +48,11 @@ const Login = () => {
       .email(errorMessages[language].value.email)
       .required(errorMessages[language].value.empty),
     password: Yup.string()
-      .matches(formRegExp.pass, errorMessages[language].value.pass)
+      .matches(formRegExp.password, errorMessages[language].value.pass)
       .required(errorMessages[language].value.empty),
     staySignedIn: Yup.bool()
   });
```
Comment (dmytro07) [inline src/pages/login/login.js:54]: Put validation schema in separate file in validators folder

Comment (Haytham1337) [reply src/pages/login/login.js]: fixed!

**Review by IlyaBelik — APPROVED** [review]

**Review by M1ndForce — APPROVED** [review]

**Review by dmytro07 — APPROVED** [review]

**Pull request merged**